srcpkgs/corosync/files/corosync.conf

```diff
@@ -0,0 +1,24 @@
+# Please read the corosync.conf.5 manual page
```

Comment:
This file should be dropped, it is not in Void's style to vendor example configurations.  Its assumed that the target market for void can read the fine manuals provided by upstream.